Tests/PersistenceBaseTest+Basic.m

```diff
@@ -16,55 +16,136 @@ -(void)basic_setupFixtures {
-            [self buildPersistenceManagerWithDefaultClient:NO];
+            id<CDAPersistedAsset> asset = [[self.persistenceManager fetchAssetsFromDataStore] firstObject];
+            XCTAssertEqualObjects(@"512_black.png", asset.url.lastPathComponent, @"");
+            id cat = [[self.persistenceManager fetchEntriesFromDataStore] firstObject];
+            XCTAssertEqualObjects(@"Test", [cat name], @"");
 
+            // "added.json"
             [self.persistenceManager performSynchronizationWithSuccess:^{
-                [self assertNumberOfAssets:1U numberOfEntries:1U];
+                XCTAssertEqual(1U, [self.persistenceManager fetchAssetsFromDataStore].count, @"");
+                XCTAssertEqual(2U, [self.persistenceManager fetchEntriesFromDataStore].count, @"");
+
+                NSDate* timestamp = [self.persistenceManager fetchSpaceFromDataStore].lastSyncTimestamp;
+                if (![[timestamp description] hasSuffix:@":00 +0000"]) {
+                    XCTAssertNotEqualObjects(self.lastSyncTimestamp, timestamp, @"");
+                }
+                self.lastSyncTimestamp = timestamp;
+
+
                 [self buildPersistenceManagerWithDefaultClient:NO];
 
+                // "deleted.json"
                 [self.persistenceManager performSynchronizationWithSuccess:^{
-                    [self assertNumberOfAssets:2U numberOfEntries:1U];
+                    XCTAssertEqual(1U, [self.persistenceManager fetchAssetsFromDataStore].count, @"");
+                    XCTAssertEqual(1U, [self.persistenceManager fetchEntriesFromDataStore].count, @"");
+
+                    NSDate* timestamp = [self.persistenceManager fetchSpaceFromDataStore].lastSyncTimestamp;
+                    if (![[timestamp description] hasSuffix:@":00 +0000"]) {
+                        XCTAssertNotEqualObjects(self.lastSyncTimestamp, timestamp, @"");
+                    }
+                    self.lastSyncTimestamp = timestamp;
                     [self buildPersistenceManagerWithDefaultClient:NO];
 
+                    // "added-asset.json"
                     [self.persistenceManager performSynchronizationWithSuccess:^{
-                        [self assertNumberOfAssets:1U numberOfEntries:1U];
+                        XCTAssertEqual(2U, [self.persistenceManager fetchAssetsFromDataStore].count, @"");
+                        XCTAssertEqual(1U, [self.persistenceManager fetchEntriesFromDataStore].count, @"");
+                        NSDate* timestamp = [self.persistenceManager fetchSpaceFromDataStore].lastSyncTimestamp;
+                        if (![[timestamp description] hasSuffix:@":00 +0000"]) {
+                            XCTAssertNotEqualObjects(self.lastSyncTimestamp, timestamp, @"");
+                        }
+                        self.lastSyncTimestamp = timestamp;
                         [self buildPersistenceManagerWithDefaultClient:NO];
 
+                        // "deleted-asset.json"
```

Comment:
it's just a comment letting me know which Fixture file is being used that I added so i could follow the test.